### PR TITLE
Fix paths in plot_meg_sensors.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/3056

### CODE REVIEW

File: examples/visualization/plot_meg_sensors.py [hunk 1]
```diff
@@ -14,23 +14,24 @@
...
 from mne.viz import plot_trans
 
 print(__doc__)
 
-bti_path = op.abspath(op.dirname(mne.__file__)) + '/io/bti/tests/data/'
-kit_path = op.abspath(op.dirname(mne.__file__)) + '/io/kit/tests/data/'
+bti_path = op.join(op.abspath(op.dirname(__file__)), '..', '..', 'mne', 'io',
+                   'bti', 'tests', 'data')
+kit_path = op.join(op.abspath(op.dirname(__file__)), '..', '..', 'mne', 'io',
+                   'kit', 'tests', 'data')
```
Comment (Eric89GXL) [inline examples/visualization/plot_meg_sensors.py]: Same comment as @agramfort made before about these paths -- we shouldn't assume that the `example` file is still in its original location, so `mne.__file__` shoudl be more robust. What did you end up doing in your last PR to deal with this comment and get things working?

Comment (agramfort) [inline examples/visualization/plot_meg_sensors.py]: don't rely on the fact that examples are ran from the examples folder :( :(

Comment (jaeilepp) [inline examples/visualization/plot_meg_sensors.py]: I did what I commented in https://github.com/mne-tools/mne-python/issues/3056. We probably don't want to add test data to ``setup.py``. I really don't know if there is another way of pointing to the data without adding it to installation.

**Comment by jaeilepp** [comment]
Circle build fails now because of this. I see that there are three options to make this work:
1) Modify the example to not use the bti and kit data,
2) use the paths proposed in this PR or
3) add the bti and kit data to ``mne/data``.
I really don't see adding test data to ``setup.py`` as an option

**Comment by Eric89GXL** [comment]
How about
4) Figure out why `circle` has a .egg installed instead of a `.egg-link`
...? It seems like `setup.py` develop should work. Did we install with `python setup.py install` at some point and it stuck around?

**Comment by Eric89GXL** [comment]
In other words, why is `op.dirname(mne.__file__)` not `/home/ubuntu/mne-python/mne`? It's what I would expect.

**Comment by jaeilepp** [comment]
Hmm. Actually circle probably does it automatically in the beginning. We need to find a way around it.

**Comment by Eric89GXL** [comment]
A little help from `rm -Rf` should do the trick

**Comment by jaeilepp** [comment]
That should do it.

**Comment by Eric89GXL** [comment]
Let's find out

**Pull request merged**